quaistats/quaistats.go

```diff
@@ -945,3 +996,20 @@ func (s *Service) reportPeers(conn *connWrapper) error {
 	}
 	return conn.WriteJSON(report)
 }
+
+// dirSize returns the size of a directory in bytes.
+func dirSize(path string) (int64, error) {
+	var size int64
+	err := filepath.WalkDir(path, func(_ string, d fs.DirEntry, err error) error {
```

Comment:
Walking the directory tree and accumulating the size of each file could get expensive, especially to do on every block. Wouldn't it be better to let the OS tell you the size of the directory?